integreat_cms/api/v3/locations.py

```diff
@@ -3,6 +3,7 @@
 """
 from django.conf import settings
 from django.http import JsonResponse
+from distutils.util import strtobool
```

Comment:
```suggestion
from distutils.util import strtobool
from django.conf import settings
from django.http import JsonResponse
```
This should fix the pylint warning